docs/variants/msi_z790/releases.md

```diff
@@ -13,6 +13,112 @@ For details about our release process please read
+- [Updated ME to the newer version; refer to SBOM section below](https://github.com/Dasharo/dasharo-blobs/tree/main/msi/ms7e06)
+- [Updated Flash Descriptor to unlock regions; refer to SBOM section below](https://github.com/Dasharo/dasharo-blobs/tree/main/msi/ms7e06)
+- [Switched to the Raptor Lake-S Client FSP; refer to SBOM section below](https://github.com/intel/FSP/tree/481ea7cf0bae0107c3e14aa746e52657647142f3/RaptorLakeFspBinPkg/Client/RaptorLakeS)
+- Get SMBIOS serial number and UUID from ROMHOLE region instead of CBFS
+  (ROMHOLE is preserved during FlashBIOS and updates via DTS)
+- Rebased coreboot on 4.21 tag
+- Reduced the amount of microcode blobs included in the build, due to
+  redundancy (some blobs had the same shasum and supported the same CPUIDs)
+- Enroll default UEFI Secure Boot keys on the first boot
+- [Improved UEFI Secure Boot menu user experience](https://docs.dasharo.com/dasharo-menu-docs/device-manager/#secure-boot-configuration)
+- DDR4 variant SMBIOS product name to reflect MSI naming
+- Scope of reset to default hotkey to global in firmware setup
+- Booting DTS over iPXE via HTTPS
+- Removed the i225/i226 EFI driver from the builds in favor of native iPXE
+  driver
+- Switched iPXE repository to Dasharo fork for the native iPXE driver for
+  i225/i226
+
+### Fixed
+
+- [No audio playback via headset](https://github.com/Dasharo/dasharo-issues/issues/483)
+- [Auto Boot Time-out is reset to 0 when F9 is pressed](https://github.com/Dasharo/dasharo-issues/issues/513)
+- [Some RPL-S CPUs ( 0xB06F2 (RPL-S C0) or 0xB06F5 (RPL-S H0)) have a problem with booting on v0.9.0](https://github.com/Dasharo/dasharo-issues/issues/496)
+- [Change boot order menu is confusing](https://github.com/Dasharo/dasharo-issues/issues/422)
+- [The setup menu does not issue a reset, resulting in saved but unapplied settings](https://github.com/Dasharo/dasharo-issues/issues/398)
+- [PCI Express Resizable BAR programming](https://github.com/Dasharo/dasharo-issues/issues/565)
+- [PCI Express resource allocation for Intel ARC A750](https://github.com/Dasharo/dasharo-issues/issues/584)
+- [CPU frequency not displayed on setup front page](https://github.com/Dasharo/dasharo-issues/issues/662)
+- P2SB BAR not properly reserved in ACPI
+- Power LED not blinking during S3 sleep
+- PS/2 controller not enabled at first boot after flashing
+- Incorrect C-states reported in ACPI unsupported by HW
+- Incorrect first timestamp format in ACPI FPDT
+
+### Known issues
+
+- [Windows 11 installer unable to detect i225 Ethernet NIC](https://github.com/Dasharo/dasharo-issues/issues/482)
+- [Cannot wake from suspend via RTC on QubesOS](https://github.com/Dasharo/dasharo-issues/issues/484)
+- [Windows 11 VBS (Virtualization-based Security) appears Not enabled on System Information](https://github.com/Dasharo/dasharo-issues/issues/539)
+- [No ability to change active PCR banks with TPM PPI in FW](https://github.com/Dasharo/dasharo-issues/issues/521)
+
```

Comment:
@miczyg1 please add one last known issue to release notes: https://github.com/Dasharo/dasharo-issues/issues/683

Reply:
Added: 2c3298eacb9deff6c99330f7b03746d654172a7d